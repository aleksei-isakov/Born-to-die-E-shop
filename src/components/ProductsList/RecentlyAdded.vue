<template>
  <div class="recently-added">{{ text }}</div>
</template>

<script>
export default {
  name: 'RecentlyAdded',

  props: {
    text: {
      type: String,
      default: '',
      required: true
    }
  }
};
</script>

<style scoped>
.recently-added {
  text-align: start;
  margin-bottom: 20px;
  color: #8b8a8a;
  font-size: 16px;
}
</style>
