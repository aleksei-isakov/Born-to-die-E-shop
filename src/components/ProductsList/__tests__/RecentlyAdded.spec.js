import { mount } from '@vue/test-utils';
import RecentlyAdded from '@/components/ProductsList/RecentlyAdded.vue';

describe('RecentlyAdded', () => {
  it('renders correctly', () => {
    const wrapper = mount(RecentlyAdded, {
      propsData: {
        text: 'Recently added'
      }
    });

    expect(wrapper.element).toMatchSnapshot();
  });
});
