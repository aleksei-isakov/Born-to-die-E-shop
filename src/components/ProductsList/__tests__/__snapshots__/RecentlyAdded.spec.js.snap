// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentlyAdded renders correctly 1`] = `
<div
  class="recently-added"
>
  Recently added
</div>
`;
